yb-voyager/cmd/common.go

```diff
@@ -924,6 +924,12 @@ func getImportedSnapshotRowsMap(dbType string) (*utils.StructMap[sqlname.NameTup
 	return snapshotRowsMap, nil
 }
 
+func safeDereferenceInt64(ptr *int64) int64 {
+	if ptr != nil {
+		return *ptr
+	}
+	return 0
+}
```

Comment:
For now it is fine, but how about moving it to `utils.go` so that it can be used anywhere in future, no cyclic dependency issue will be there.